pkg/protocol/engine/mempool/transaction.go

```diff
@@ -7,9 +7,13 @@ import (
 type SignedTransaction interface {
 	// ID returns the identifier of the Transaction that contains a signature.
 	ID() (iotago.SignedTransactionID, error)
+	// MustID works like ID but panics if the SignedTransactionID can't be computed.
+	MustID() iotago.SignedTransactionID
 }
 
 type Transaction interface {
 	// ID returns the identifier of the Transaction.
 	ID() (iotago.TransactionID, error)
+	// MustID works like ID but panics if the TransactionID can't be computed.
+	MustID() iotago.TransactionID
 }
```

Comment:
It seems more reasonable to put these interfaces in iota.go.  :thinking: 
Did Transaction and SignedTransaction structure in iota.go implement these interfaces? If yes, then we also need to implement MustID() for each.